openprescribing/frontend/tests/test_api_utils.py

```diff
@@ -12,3 +12,12 @@ def do_long_running_query():
                 cursor = connection.cursor()
                 cursor.execute("select pg_sleep(0.01);")
         self.assertRaises(OperationalError, do_long_running_query)
+
+    def test_param_to_list(self):
+        from api.view_utils import param_to_list
+
+        self.assertEquals(param_to_list('foo'), ['foo'])
+        self.assertEquals(param_to_list('foo,bar'), ['foo', 'bar'])
+        self.assertEquals(param_to_list('foo,bar'), ['foo', 'bar'])
```

Comment:
This test is identical to the one above. Was it supposed to test something else, or is an accidental dupe?